static/app/views/monitors/components/monitorQuickStartGuide.tsx

```diff
@@ -127,6 +127,16 @@ const onboardingGuides: Record<string, OnboardingGuide> = {
   },
 };
 
+/**
+ * The platforms that are well supported for crons
+ */
+export const platformsWithGuides = Array.from(
+  Object.values(onboardingGuides).reduce((combinedPlatforms, guide) => {
+    guide.platforms?.forEach(platform => combinedPlatforms.add(platform));
+    return combinedPlatforms;
+  }, new Set())
```

Comment:
would it be better to use spread here and avoid the forEach + return?
```suggestion
  Object.values(onboardingGuides).reduce((combinedPlatforms, guide) => (
    new Set([...combinedPlatforms, ...(guide.platforms ?? [])])
  ), new Set())
```

Reply:
the return is slightly better since we're not creating a new set everytime I guess. Sort of indifferent

Reply:
thats fair, sgtm